tools/pytorch2onnx.py

```diff
@@ -119,6 +144,10 @@ def parse_args():
         nargs='+',
         default=[224, 224],
         help='input image size')
+    parser.add_argument(
+        '--dynamic-shape',
+        action='store_true',
+        help='export dynamic onnx graph')
```

Comment:
Change to `Whether to export ONNX with dynamic input shape. Defaults to False.`

Reply:
right!